rust/sdk-processor/src/steps/common/processor_status_saver.rs

```diff
@@ -19,6 +20,7 @@ use processor::schema::{backfill_processor_status, processor_status};
 pub fn get_processor_status_saver(
     conn_pool: ArcDbPool,
     config: IndexerProcessorConfig,
+    is_parquet: bool,
```

Comment:
discussed offline: consider using a more descriptive enum instead of this bool flag. Ideally we are able to determine what type of `processor_status_saver` to return just from the config. It's weird/inconsistent that we have this function determine which enum to return while relying on a user input to help with that decision